executorlib/interactive/shared.py

```
@@ -19,6 +22,37 @@
 
 
 class ExecutorBroker(ExecutorBase):
+    def submit(self, fn: callable, *args, resource_dict: dict = {}, **kwargs) -> Future:
```

Ruff

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)